### ISSUE TITLE
Applying the Stamp Brush tool quickly can result in the blue mask image sticking around

### ISSUE THREAD

**Issue by mitchcurtis** [bjorn/tiled#1569]
![tiled-fast](https://cloud.githubusercontent.com/assets/6002340/26032588/42b6490a-3897-11e7-94c7-78a4d02cddc5.gif)

**Comment by bjorn** [comment]
Yeah, I've noticed this as well and this kind of issue has been around since forever, though I think usually you'd have to scroll as well to cause it so maybe not exactly in this form.

I think it's probably an issue in QGraphicsView but I have so far not entirely ruled out that it may be an issue in Tiled. Regardless, it would be good to find a fix for this...
